src/apprt/gtk/Window.zig

```diff
@@ -99,6 +99,7 @@ pub fn init(self: *Window, app: *App) !void {
     self.window = gtk_window;
     c.gtk_window_set_title(gtk_window, "Ghostty");
     c.gtk_window_set_default_size(gtk_window, 1000, 600);
+    c.gtk_widget_add_css_class(@ptrCast(gtk_window), "ghostty");
```

Comment:
Hmm, I wonder if a more specific CSS name would be more useful.

Reply:
I agree, I think `terminal-window` may be better (because there may be other types of windows in the future).